blueprint/core/blueprint_View.h

```diff
@@ -72,6 +72,25 @@ namespace blueprint
         /** Dispatches a mouseDoubleClick event to the React application. */
         void mouseDoubleClick (const juce::MouseEvent& e) override;
 
+        /** Dispatches a keyPress event to the React application. */
+        bool keyPressed (const juce::KeyPress& e) override;
+
+        //==============================================================================
+        /** Invokes, if exists, the respective view event handler. */
+        template <typename... Ts>
+        void dispatchViewEvent (const juce::String& eventType, Ts... args)
+        {
+            JUCE_ASSERT_MESSAGE_THREAD
+
+            if (props.contains(eventType) && props[eventType].isMethod())
+            {
+                std::vector<juce::var> vargs { args... };
```

Comment:
You can use std::array here if you wish:

```
std::array<juce::var, sizeof...(args)> vargs = { args... };
```

Reply:
Ah, that's nice! Will do